pom.xml

```diff
@@ -31,7 +31,7 @@
 	<packaging>hpi</packaging>
 	<name>Micro Focus Application Automation Tools</name>
 	<description>The plugin integrates Jenkins with the following Micro Focus products - Service Virtualization,
-		LoadRunner, Performance Center, Unified Functional Testing, QuickTest Professional, Service Test, Mobile Center,
+		LoadRunner, Performance Center, Unified Functional Testing, QuickTest Professional, Service Test, UFT Mobile,
```

Comment:
All the products in the description are not abbreviated, I would say Unified Functional Testing Mobile instead.